### PR TITLE
Set up acceptance testing

### PR DESCRIPTION
**Pull request by akxcv**
It's impossible to run Jest in the browser, so we'll have to use Jasmine for acceptance testing. I considered moving completely away from Jest to Jasmine for unit testing, too, but Jest is just better for that.
To sum it up, now we use Jest + Enzyme for unit testing and Karma + Jasmine for acceptance testing.
Travis only comes with Firefox, so we should consider using Circle CI to run acceptance tests in different browsers.
Moreover, Travis' Firefox is *very old* (v31). That's why we have to use `babel-polyfill` in our acceptance tests.

### CODE REVIEW

**Comment by akxcv** [comment]
Looks like Travis uses Firefox v31, which doesn't fully support ES6. Probably Karma setup is wrong and Babel doesn't kick in or something.

File: test/acceptance/animated.test.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    jasmine.clock().tick(10000)
+    expect(component).toDisplayDigits('00')
+    jasmine.clock().tick(10000)
+    expect(component).toDisplayDigits('00')
+  })
+
+  it('changes with non-zero easing time', function (done) {
+    if (process.env.WATCH) {
+      pending(
+        `This spec in too inconsistent for the watch mode and thus is disabled.
```
Comment (JelF) [inline test/acceptance/animated.test.js]: What does inconsistenence mean here? If it is 100% consisted outside WATCH environment, it should better be reworded

File: test/acceptance/static.test.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import Counter from '../../src/'
```
Comment (JelF) [inline test/acceptance/static.test.js]: may "abosolute" (related from source root) paths be used here?

File: test/acceptance/animated.test.js (hunk 1 above)
Comment (akxcv) [inline test/acceptance/animated.test.js]: It's not 100% consistent outside watch mode, either :(
To be honest, I'm tempted to remove this spec completely, but it feels wrong not to test with non-zero transition time.

File: test/acceptance/static.test.js (hunk 2 above)
Comment (akxcv) [inline test/acceptance/static.test.js]: Simply `/` would reference the root of one's *file system*. There are workarounds, but they are a little too elaborate for this project, in my opinion.

**Pull request merged**